src/sections/strategy.html

```diff
@@ -0,0 +1,25 @@
+<div class="lg:px-36 px-10 md:px-20 relative">
+    <div class="w-full flex justify-center sticky top-0 z-20">
+        <img src="/images/small-yellow-shape.svg" alt="" class=" lg:w-[18%]  w-[60%] md:w-[25%]">
```

Comment:
Avoid empty attributes and unnecessary spaces in your codebase
```suggestion
        <img src="/images/small-yellow-shape.svg" alt="yellow shaped figure" class="lg:w-[18%]  w-[60%] md:w-[25%]">
```